manim/mobject/mobject.py

```diff
@@ -2287,6 +2287,28 @@ def construct(self):
             sm1.interpolate_color(sm1, sm2, 1)
         return self
 
+    def match_points(self, mobject: "Mobject", copy_submobjects: bool = True):
+        """Edit points, positions, and submobjects to be identical
+        to another :class:`~.Mobject`, while keeping the style unchanged.
+
+        Examples
+        --------
+        .. manim:: BecomeScene
+
+            class MatchPointsScene(Scene):
+                def construct(self):
+                    circ = Circle(fill_color=RED, fill_opacity=0.8)
+                    square = Square(fill_color=BLUE, fill_opacity=0.2)
+                    self.add(circ)
+                    self.wait(0.5)
+                    circ.match_points(square)
```

Comment:
Wouldn't it be better if we actually show the transformation?
```suggestion
                    self.play(circ.animate.match_points(square))
```

Reply:
I was trying to match the `become` example, but I agree that this is better.